src/compas/geometry/primitives/shapes/cone.py

```diff
@@ -167,6 +171,35 @@ def volume(self):
         """Float: The volume of the cone."""
         return pi * self.circle.radius**2 * (self.height / 3)
 
+    def to_vertices_and_faces(self, **kwargs):
+        """Returns a list of vertices and faces, called by `Mesh.from_shape()`."""
```

Comment:
```suggestion
        """Returns a list of vertices and faces, called by ``Mesh.from_shape()``."""
```

Reply:
good to know `code` formatting needs double "`"s, but this specific part will probably removed anyway as suggested by @tomvanmele